catalog/app/containers/Package/index.js

```diff
@@ -168,14 +170,18 @@ const Install = ({ name, owner }) => (
       <FormattedMessage {...strings.installThen} />
     </p>
     <Code>
-      <Unselectable>$ </Unselectable>quilt install {owner}/{name}
+      <Unselectable>$ </Unselectable>quilt install {makeHandle(owner, name)}
     </Code>
     <p><FormattedMessage {...strings.sell} /></p>
     <Help href={blogManage} />
     <h2><FormattedMessage {...strings.access} /></h2>
     <Tabs>
       <Tab label="Python">
-        <Code>from quilt.data.{owner} import {name}</Code>
+        {
+          config.team ?
```

Comment:
This happens to work - but if we were to run the stage or prod catalog in ECS, we'd have `team: {name: ''}` just cause of the way we set it in config.js.tmpl.

Reply:
that is kinda' dangerous. there are other places in this PR that would break since `{}` is truthy. i can always cue off of `config.team.name` (a bit ugly) or can we unset this if it's `''`? i don't suppose cloud formation has nullable strings?

Reply:
fixed in `static/config.js`; we can never have an empty team id and still have a `team` object.